nexuse2e-webapp/src/main/frontend/src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <title>NEXUSe2e</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <link rel="icon" type="image/x-icon" href="favicon.ico"/>
+  <link href="https://fonts.gstatic.com" rel="preconnect">
```

Comment:
Are we sure, that in every Setup the browser has internet access? Can we include this in the assets?

Reply:
Removed line 8